packages/eslint-plugin/src/rules/no-unused-vars.ts

```diff
@@ -419,9 +419,7 @@ export default util.createRule<Options, MessageIds>({
 
         const unusedVars = collectUnusedVariables();
 
-        for (let i = 0, l = unusedVars.length; i < l; ++i) {
```

Comment:
This wasn't reported by the rule. Should it be?

Reply:
Heh probably. I forgot folks used to optimize loops this way!